DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/ViewDelegateControllerTests.cs

```diff
@@ -67,5 +67,41 @@ public void Deactivating_delegate_on_wrong_centre_returns_not_found_result()
             // then
             result.Should().BeNotFoundResult();
         }
+
+
+        [Test]

```

Comment:
Just noticed another stray empty line here that can be removed so then we only have 1 line between tests.